### ISSUE TITLE
DeepGS: Predicting phenotypes from genotypes using Deep Learning

### ISSUE THREAD

**Issue by SiminaB** [greenelab/deep-review#743]
https://doi.org/10.1101/241414 https://www.biorxiv.org/content/early/2017/12/31/241414

> Motivation: Genomic selection (GS) is a new breeding strategy by which the phenotypes of quantitative traits are usually predicted based on genome-wide markers of genotypes using conventional statistical models. However, the GS prediction models typically make strong assumptions and perform linear regression analysis, limiting their accuracies since they do not capture the complex, non-linear relationships within genotypes, and between genotypes and phenotypes. Results: We present a deep learning method, named DeepGS, to predict phenotypes from geno-types. Using a deep convolutional neural network, DeepGS uses hidden variables that jointly repre-sent features in genotypic markers when making predictions; it also employs convolution, sampling and dropout strategies to reduce the complexity of high-dimensional marker data. We used a large GS dataset to train DeepGS and compare its performance with other methods. In terms of mean normalized discounted cumulative gain value, DeepGS achieves an increase of 27.70%~246.34% over a conventional neural network in selecting top-ranked 1% individuals with high phenotypic values for the eight tested traits. Additionally, compared with the widely used method RR-BLUP, DeepGS still yields a relative improvement ranging from 1.44% to 65.24%. Through extensive simulation experiments, we also demonstrated the effectiveness and robustness of DeepGS for the absent of outlier individuals and subsets of genotypic markers. Finally, we illustrated the complementarity of DeepGS and RR-BLUP with an ensemble learning approach for further improving prediction performance. Availability: DeepGS is provided as an open source R package available at https://github.com/cma2015/DeepGS.
